docs/UsersGuide/source/ConfigWorkflow.rst

```diff
@@ -1774,11 +1779,14 @@ Typically, the following parameters must be set explicitly by the user in the co
    The starting forecast hour for the plotting task. For example, if a forecast starts at 18h/18z, this is considered the 0th forecast hour, so "starting forecast hour" should be 0, not 18. If a forecast starts at 18h/18z, but the user only wants plots from the 6th forecast hour on, "starting forecast hour" should be 6.
 
 ``PLOT_FCST_INC``: (Default: 3)
-   Forecast hour increment for the plotting task. This may be the same as ``INCR_CYCL_FREQ``, or it may be a multiple of ``INCR_CYCL_FREQ``. For example, if ``INCR_CYCL_FREQ`` is set to 3, there will be forecast output every three hours for the duration of the forecast. If the user wants plots of all of this output, they should set ``PLOT_FCST_INC: 3``. If the user only wants plots for some of the output (e.g., every 6 hours), they should set ``PLOT_FCST_INC: 6``. However, there must be forecast output available at the designated increments to produce the plots. In this example, setting ``PLOT_FCST_INC: 7`` would produce an error because there is only forecast output available for hours 3, 6, 9, ..., etc. 
+   Forecast hour increment for the plotting task. This may be the same as ``LBC_SPEC_INTVL_HRS``, or it may be a multiple of ``LBC_SPEC_INTVL_HRS``. For example, if ``LBC_SPEC_INTVL_HRS`` is set to 3, there will be forecast output every three hours for the duration of the forecast. If the user wants plots of all of this output, they should set ``PLOT_FCST_INC: 3``. If the user only wants plots for some of the output (e.g., every 6 hours), they should set ``PLOT_FCST_INC: 6``. However, there must be forecast output available at the designated increments to produce the plots. In this example, setting ``PLOT_FCST_INC: 7`` would produce an error because there is only forecast output available for hours 3, 6, 9, ..., etc. 
```

Comment:
@gspetro I think (correct me if I'm wrong) `LBC_SPEC_INTVL_HRS` is getting mixed up here with the forecast output frequency (i.e. how often the `dynf${hhh}.nc` and `phyf${hhh}.nc` files are output).  The latter is set by either one of the parameters `output_fh` (**output** **f**requency in **h**ours) and `nsout` (**n**umber of (time) **s**teps per **out**put).  Currently, `output_fh` and `nsout` are hard-coded in the script `ush/create_model_configure_file.py`, so the user can't set them (we should really allow the user to change this).  `nsout` is only used if the user has activated `subhourly_post`, which they can't really do anyway since it's broken.  So the output frequency is really just set by `output_fh`, which is hard-coded to 1.  Then the smallest value the user can set `PLOT_FCST_INC` to is `outout_fh`=1. 
 `LBC_SPEC_INTVL_HRS` is just the time interval with which LBCs are read in (so having it smaller will give a better forecast), but it doesn't affect the frequency of the output files and so shouldn't affect `PLOT_FCST_INC``.  So setting `LBC_SPEC_INTVL_HRS` to 3 will still give forecast output every 1 hour since `output_fh` is still 1.

Reply:
@gsketefian Thanks for the clarification. I was definitely mixing up LBC_SPEC_INTVL_HRS and forecast output frequency. I think I've fixed it now.